Index.html

```diff
@@ -24,17 +24,17 @@
 
     <section>
       <article>
-        <a href="academichallfire.html"
+        <a href="academichallfire.html">
         <header>
           <H2>The Academic Hall Fire</H2>
-        </header>
+        </header> 
         <p>On January 9, 1892, Academic Hall, the institution's main building, burned in a fire that completely gutted the building, leaving little more standing than six stone Ionic columns. Under the administration of Missouri Governor David R. Francis, the university was rebuilt, with additions that shaped the modern institution. On January 9, 1892, Academic Hall, the institution's main building, burned in a fire that completely gutted the building, leaving little more standing than six stone Ionic columns. Under the administration of Missouri Governor David R. Francis, the university was rebuilt, with additions that shaped the modern institution. (Wikipedia)</p>
          <img src="IMG/academic_hall_fire.jpg" alt="Picture of the Academic Hall fire" height="200" width="200">
```

Comment:
Really glad to see that you're remembering your alt text here, and also that you're starting to think about the image paths/asset paths for your photos. A couple of notes: it's best practice to make sure that the asset path (here, the folder called IMG) is all in lowercase; and, keep in mind that the width/height of the photo need to be the literal width/height of the photo if it's not to skew. In other words, if you're going to have the photo that square shape, make sure it's actually square before you size it there.